hw1/task1_2_pipeline.js

```diff
@@ -0,0 +1,16 @@
+const fs = require('fs');
+const csv=require('csvtojson');
+const { pipeline } = require('stream');
```

Comment:
You are importing pipeline method but not using it. 
So you can use it like this:

```
pipeline(
  fs.createReadStream(csvFilePath),
  ({
    output: "json",
    delimiter: ';'
  }),
 fs.createWriteStream('./hw1/csv/task1_2/text_out_pipe.txt'),
 (error) => {...}
);
```